### ISSUE TITLE
delete a filter

### ISSUE THREAD

**Issue by alagahdejesus** [IDEMSInternational/R-Instat#8066]
**Is your feature request related to a problem? Please describe.**

I have created a filter and i wanna delete it now. How to do? it must have a button to do this.

**Describe the solution you'd like**

The solution is to have the possibilite to delete filter when it work finish and the dataset is deleted.

**Describe alternatives you've considered**

**Additional context**

here the filter is filter_year

![image](https://user-images.githubusercontent.com/29559527/213460088-9bb41303-ae1e-4d8b-aebd-f59e9a8a4118.png)

**Comment by derekagorhom** [comment]
![image](https://user-images.githubusercontent.com/76208189/213635654-999e85b2-f37b-452d-ad48-1ecc7aacde19.png)

Hello @alagahdejesus if you want to remove a filter;

Right click on the dataframe > `remove current filter`. This will remove the filter from your dataset.
Hope this helps

**Comment by alagahdejesus** [comment]
No, No. Not this. I want to delete it from thé list of filters. There no option for this. 

Envoyé depuis Yahoo Mail pour Android 
 
  Le ven., janv. 20, 2023 à 7:57, Derrick ***@***.***> a écrit:   




Hello @alagahdejesus if you want to remove a filter;

Right click on the dataframe > remove current filter. This will remove the filter from your dataset.
Hope this helps

**Comment by rdstern** [comment]
@alagahdejesus that's a good question.  In R-Instat - unlike ordinary R - a filter is stored as an object.  So use this menu option:

![image](https://user-images.githubusercontent.com/11226469/213675217-6bf00cd6-7041-4a30-87a6-89d67095c97e.png)

In the dialogue, you can specify filters and then delete them.

**Comment by alagahdejesus** [comment]
Waouh ! Thanks a lot You help me so much. 
Envoyé depuis Yahoo Mail pour Android 
 
  Le ven., janv. 20, 2023 à 11:37, ***@***.***> a écrit:   


@alagahdejesus that's a good question. In R-Instat - unlike ordinary R - a filter is stored as an object. So use this menu option:



In the dialogue, you can specify filters and then delete them.
